src/provider/match.js

```diff
@@ -46,13 +46,15 @@ const match = (id, source, data) => {
 
 const check = (url) => {
 	const song = { size: 0, br: null, url: null, md5: null };
-	let header = { range: 'bytes=0-8191' };
+	let header = { range: 'bytes=0-8191', 'accept-encoding': 'identity' };
```

Comment:
See https://datatracker.ietf.org/doc/html/rfc2616#section-3.5 : `identity ` means that there shall be no transformation whatsoever when checking audio source. Some CDN refuses to send `content-length` under streaming mode.